assets/js/components/post-searcher.js

```diff
@@ -46,12 +47,14 @@ import Layout from './layout/layout';
 class PostSearcher extends Component {
 	constructor( props ) {
 		super( props );
-
+		this.disallowedSelections = [ 'No results found' ];
```

Comment:
This is a translated string, so this reference must also be. We can probably just extract the string itself to a property as well so all usage references same value rather than repeating it multiple times in the component.

Reply:
Updated.